src/app/update/update.page.ts

```diff
@@ -216,17 +215,17 @@ onFirmwareFileSelected(event: Event): void {
       this.totalSize >> 16 & 0xFF,
       this.totalSize >> 8 & 0xFF,
       this.totalSize & 0xFF
```

Comment:
**Suggestion:** Use a constant for the magic number '256' to improve code readability.
```suggestion
      const BYTE_SIZE = 256;
      this.totalSize >> 16 & BYTE_SIZE,
      this.totalSize >> 8 & BYTE_SIZE,
      this.totalSize & BYTE_SIZE
```

Reply:
won't do